bower.json

```diff
@@ -1,20 +1,35 @@
 {
   "name": "html-imports",
   "main": "html-imports.min.js",
+  "description": "HTML Imports polyfill",
   "version": "1.0.0-rc.2",
-  "homepage": "http://webcomponents.org",
+  "homepage": "https://webcomponents.org/polyfills",
   "authors": [
-    "The Polymer Authors"
+    "The Polymer Authors (https://polymer.github.io/AUTHORS.txt)"
   ],
   "repository": {
     "type": "git",
     "url": "https://github.com/webcomponents/html-imports.git"
   },
   "keywords": [
-    "webcomponents"
+    "html-imports",
+    "htmlimports",
+    "web-components",
+    "webcomponents",
+    "polyfill",
+    "shim"
   ],
   "license": "BSD-3-Clause",
-  "ignore": [],
+  "ignore": [
+    "**/.*",
+    "node_modules",
+    "bower_components",
+    "test",
+    "tests",
+    "src",
+    "gulpfile.js",
+    "package.json"
```

Comment:
Ok, done.

BTW, you could do it yourself, because all my pull requests allow edits from maintainers. ;)